starknet_py/utils/contructor_args_translator.py

```diff
@@ -0,0 +1,33 @@
+# pyright: reportGeneralTypeIssues=false
+
+from typing import List
+
+from starkware.starknet.public.abi_structs import identifier_manager_from_abi
+
+from starknet_py.utils.data_transformer import FunctionCallSerializer
+
+
+def translate_constructor_args(abi: List, constructor_args: any) -> List[int]:
+    constructor_abi = next(
+        (member for member in abi if member["type"] == "constructor"),
+        None,
+    )
```

Comment:
Could we make a local variable here

Reply:
I'm not sure wdym 😅

Reply:
I meant to assign `(member for member in abi if member["type"] == "constructor")` to local variable, and then use it here; sorry, should've been more specific 🙈